airbyte-integrations/connectors/source-marketo/source_marketo/schemas/activity_types.json

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "integer"
```

Comment:
nit: we should avoid the strict declaring of the types, because there could be null values, even if you're 100% sure that the value is present. Make it nullable by default.  Correct this in all schemas, please.

Reply:
Updated.